tests/integ/test_tf_keras.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+
+import numpy as np
+import pytest
+
+from sagemaker.tensorflow import TensorFlow
+from tests.integ import DATA_DIR
+from tests.integ.timeout import timeout_and_delete_endpoint_by_name, timeout
+
+@pytest.mark.continuous_testing
+def test_keras(sagemaker_session, tf_full_version):
+    script_path = os.path.join(DATA_DIR, 'cifar_10', 'source')
+    dataset_path = os.path.join(DATA_DIR, 'cifar_10', 'data')
+
+    with timeout(minutes=45):
```

Comment:
Is it necessary to have a 45 minutes run for these test? Can we reduce the training steps and evaluation steps?

Reply:
Yep we need this because it currently times out frequently due to hosting. And we took their advice of using 45 mins.

The current timeout for tf cifar is also 45 mins https://github.com/aws/sagemaker-python-sdk/blob/master/tests/integ/test_tf_cifar.py#L38

And related PR is here: https://github.com/aws/sagemaker-python-sdk/pull/337

Reply:
And I made the change to reduce training and evaluation steps too. Now it uses 500 and 5 which are same as what tf cifar test uses.